src/+otp/+sanzserna/+presets/Canonical.m

```diff
@@ -0,0 +1,11 @@
+classdef Canonical < otp.sanzserna.SanzSernaProblem
+    methods
+        function obj = Canonical(m)
+            if nargin < 1
+                m = 32;
```

Comment:
Most problems use "n" or "N" for the spatial discretization. Why do we use "m" here?

Reply:
m is what the paper used.  If we rename, something descriptive like `numGridCells` might be best.